Sources/Commands.swift

```diff
@@ -242,7 +242,7 @@ public func command<A:ArgumentDescriptor, A1:ArgumentDescriptor, A2:ArgumentDesc
         BoxedArgumentDescriptor(value: descriptor5),
     ])
 
-    if parser.hasOption("help") {
+    if parser.has(option: "help") {
```

Comment:
I first removed those, but then the compiler gets confused and I had to add a bunch of typecasts. 

You get:
```swift
  public func has(_ name: Option) -> Bool {}
  public func has(_ flag: Flag) -> Bool {}
```

If I then call `parser.has("a")`, swift will get confused because `"a"` is a string. This has mostly to do with the unit tests.